### DISCUSSION TITLE
Tree View Welcome Content Support

### DISCUSSION THREAD

**Discussion by cemalgnlts** [CodinGame/monaco-vscode-api#298] [Q&A] [answered]
Hi,

Is there any support for https://code.visualstudio.com/api/extension-guides/tree-view#welcome-content?

I tried the example but the content does not appear.

**Comment by CGNonofr** [comment]
It will in https://github.com/CodinGame/monaco-vscode-api/pull/294

**Reply by cemalgnlts** [reply]
Hi, thanks for the new release. The Welcome service does not seem to work correctly (but it works in the example).

```
Uncaught TypeError: FileAccess.registerModuleContentProvider is not a function
    at welcome.js:15:12
```

https://github.com/CodinGame/monaco-vscode-api/blob/main/src/service-override/welcome.ts#L19

**Reply by CGNonofr** [reply, marked as answer]
If you are using the treemending script, you'll need to make sure it's reapplied (`npm ci` should be enough)

**Reply by cemalgnlts** [reply]
It worked :)
thanks.
